src/lib/application/application.factory.test.ts

```diff
@@ -108,5 +150,9 @@ describe('Application Factory', () => {
       '/scope-package/test/app.e2e-spec.ts',
       '/scope-package/test/jest-e2e.json',
     ]);
+
+    expect( JSON.parse(tree.readContent('/scope-package/package.json')) ).toMatchObject({
+      name: 'package',
```

Comment:
I'm not sure if this is expected

Currently (and the same behavior after this PR):

```bash
nest new @foo/bar
grep name ./foo/package.json # will be 'foo'
```

Reply:
I think it should be `@foo/bar`. Would you like to create a follow-up PR? @micalevisk

Reply:
can I fix that on this PR too?

Reply:
Sounds good!